### PR TITLE
Generate assets json wrapper

### PR DESCRIPTION
**Pull request by billwert**
The generate-assets-json script assumes you've installed the test proxy using dotnet.

For java devs, use the locally downloaded copy instead.

### CODE REVIEW

**Comment by scbedd** [comment]
FYI @billwert we are also merging this next week: https://github.com/Azure/azure-sdk-tools/pull/6300

**Comment by billwert** [comment]
Cool. I still think we need this wrapper so we can add the place our harness downloads the test proxy into to the user's PATH. I want to minimize the number of command line arguments folks have to pass if possible.

**Comment by scbedd** [comment]
> Cool. I still think we need this wrapper so we can add the place our harness downloads the test proxy into to the user's PATH. I want to minimize the number of command line arguments folks have to pass if possible.

Oh sure. Just FYI so you could adjust your plans if necessary.

File: eng/scripts/generate-assets-json-wrapper.ps1 [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+}
+
+$testProxyPath = join-path $temp "test-proxy"
+if (!(test-path $testProxyPath)) {
+    throw "Unable to find test proxy directory. Run a test proxy test to install it."
+}
+# Need the path separator for search paths, not file paths.
+$pathsep = if ($env:OS -eq "Windows_NT") { ";" } else { ":" }
+$env:Path = $env:PATH + $pathsep + $testProxyPath
+Invoke-Expression "$(join-path $PSScriptRoot ../common/testproxy/transition-scripts/generate-assets-json.ps1) -TestProxyExe Azure.Sdk.Tools.TestProxy $args"
```
Comment (joshfree) [inline eng/scripts/generate-assets-json-wrapper.ps1:14]: missing newline character

**Comment by joshfree** [comment]
@billwert is this PR still needed?

**Comment by billwert** [comment]
@samvaity up to you. Has this been a pain point for anyone else?

**Review by samvaity — APPROVED** [review]

**Comment by joshfree** [comment]
> Has this been a pain point for anyone else?

It cant hurt to have it; my comment was mostly about the PR being open for so many weeks - I was just checking to see if it was going to be merged or not

File: eng/scripts/generate-assets-json-wrapper.ps1 (hunk 1 above)
Comment (billwert) [inline eng/scripts/generate-assets-json-wrapper.ps1:14]:
```suggestion
Invoke-Expression "$(join-path $PSScriptRoot ../common/testproxy/transition-scripts/generate-assets-json.ps1) -TestProxyExe Azure.Sdk.Tools.TestProxy $args"

```

**Pull request merged**